opentreemap/treemap/js/src/streetView.js

```diff
@@ -31,15 +31,21 @@ exports.create = function(options) {
             if (status == google.maps.StreetViewStatus.OK) {
                 if (panorama === null) {
                     panorama = new google.maps.StreetViewPanorama(div, {
-                        position:pos,
+                        position: pos,
                         addressControl: showAddress,
                         panControl: false,
                     });
                 } else {
                     panorama.setPosition(pos);
                 }
-            }
-            else {
+                if (data !== null) {
+                    var cameraPos = data.location.latLng,
```

Comment:
it took a verbal explanation to understand how this works, implies some knowledge about the google street (the concept of "camera position"). please consider adding a comment to the source code explaining at a high level what this does.